packages/babel-core/src/config/option-manager.js

```diff
@@ -72,10 +77,14 @@ const ALLOWED_PLUGIN_KEYS = new Set([
   "inherits",
 ]);
 
-export default function manageOptions(opts: {}): {
-  options: Object,
-  passes: Array<Array<[ Plugin, ?{} ]>>,
-}|null {
+export default function manageOptions(
+  opts: {},
+):
```

Comment:
You could define the type in a variable just above and have a reference here, so you don't need to have this weird inline.